Python/Product/Debugger/Debugger/DebugEngine/AD7Thread.cs

```diff
@@ -126,6 +124,7 @@ class AD7Thread : IDisposable, IDebugThread2, IDebugThread100 {
                 props.dwFields |= enum_THREADPROPERTY_FIELDS.TPF_STATE;
             }
             if ((dwFields & enum_THREADPROPERTY_FIELDS.TPF_PRIORITY) != 0) {
+                // TODO: Localization
                 props.bstrPriority = "Normal";
```

Comment:
I believe so, but we should be able to check whether other languages localize these values in the Threads window.